tools/bin/automatic_migration_acceptance_test.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+. tools/lib/lib.sh
+
+assert_root
+
+echo "Starting app for automaticMigrationAcceptanceTest"
+
+
+ENV_VERSION=$(grep VERSION .env | xargs)
+ENV_VERSION=${ENV_VERSION#*=}
```

Comment:
I thought we discussed and thought we could avoid adding this file and call the task directly in the gradle file. It didn't work?